_posts/2021-05-14-bring-gitops-to-your-functions-with-argocd-part-2-the-missing-piece-tekton.md

```diff
@@ -0,0 +1,612 @@
+clusterrole.rbac.authorization.k8s.io/tekton-pipelines-webhook-cluster-access created
+role.rbac.authorization.k8s.io/tekton-pipelines-controller created
+role.rbac.authorization.k8s.io/tekton-pipelines-webhook created
+role.rbac.authorization.k8s.io/tekton-pipelines-leader-election created
+serviceaccount/tekton-pipelines-controller created
+serviceaccount/tekton-pipelines-webhook created
+clusterrolebinding.rbac.authorization.k8s.io/tekton-pipelines-controller-cluster-access created
+clusterrolebinding.rbac.authorization.k8s.io/tekton-pipelines-controller-tenant-access created
+clusterrolebinding.rbac.authorization.k8s.io/tekton-pipelines-webhook-cluster-access created
+Warning: rbac.authorization.k8s.io/v1beta1 RoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 RoleBinding
+rolebinding.rbac.authorization.k8s.io/tekton-pipelines-controller created
+rolebinding.rbac.authorization.k8s.io/tekton-pipelines-webhook created
+rolebinding.rbac.authorization.k8s.io/tekton-pipelines-controller-leaderelection created
+rolebinding.rbac.authorization.k8s.io/tekton-pipelines-webhook-leaderelection created
+customresourcedefinition.apiextensions.k8s.io/clustertasks.tekton.dev created
+customresourcedefinition.apiextensions.k8s.io/conditions.tekton.dev created
+customresourcedefinition.apiextensions.k8s.io/pipelines.tekton.dev created
+customresourcedefinition.apiextensions.k8s.io/pipelineruns.tekton.dev created
+customresourcedefinition.apiextensions.k8s.io/pipelineresources.tekton.dev created
+customresourcedefinition.apiextensions.k8s.io/runs.tekton.dev created
+customresourcedefinition.apiextensions.k8s.io/tasks.tekton.dev created
+customresourcedefinition.apiextensions.k8s.io/taskruns.tekton.dev created
+secret/webhook-certs created
+validatingwebhookconfiguration.admissionregistration.k8s.io/validation.webhook.pipeline.tekton.dev created
+mutatingwebhookconfiguration.admissionregistration.k8s.io/webhook.pipeline.tekton.dev created
+validatingwebhookconfiguration.admissionregistration.k8s.io/config.webhook.pipeline.tekton.dev created
+clusterrole.rbac.authorization.k8s.io/tekton-aggregate-edit created
+clusterrole.rbac.authorization.k8s.io/tekton-aggregate-view created
+configmap/config-artifact-bucket created
+configmap/config-artifact-pvc created
+configmap/config-defaults created
+configmap/feature-flags created
+configmap/config-leader-election created
+configmap/config-logging created
+configmap/config-observability created
+configmap/config-registry-cert created
+deployment.apps/tekton-pipelines-controller created
+service/tekton-pipelines-controller created
+horizontalpodautoscaler.autoscaling/tekton-pipelines-webhook created
+deployment.apps/tekton-pipelines-webhook created
+service/tekton-pipelines-webhook created
+```
+
+Let's install _Tekton Trigger_.
+
+```bash
+$ kubectl apply -f https://storage.googleapis.com/tekton-releases/triggers/latest/release.yaml
```

Comment:
All the output is unnecessary.

You haven't defined anywhere what "_Tekton Trigger_" is and what the differences are. Please do that.